src/main/java/frc/robot/commands/Intake2020LowerArmCMD.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.Intake2020Arm;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Intake2020LowerArmCMD extends CommandBase {
+  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
+  private final Intake2020Arm intakeArm;
+
+  /**
+   * Creates a new Intake2020LowerArmCMD.
+   *
+   * @param subsystem The subsystem used by this command.
```

Comment:
There is no parameter named `subsystem`. Did you mean `intakeArm`? This applies to all of the command classes that do the same thing.